test/test_bwd.py

```diff
@@ -6,9 +6,13 @@
 import torch
 import torch.distributed as dist
 
-from pippy.IR import pipe_split, pipeline
-from pippy.PipelineSchedule import Schedule1F1B, ScheduleGPipe
-from pippy.PipelineStage import PipelineStage
+from pippy import (
```

Comment:
does this cleanup align with how things will work after you move the tests into torch? will it just be s/pippy/torch.something.something/?

Reply:
Yes.